src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheck.java

```diff
@@ -52,6 +52,17 @@
  * <pre>
  * &lt;module name="LocalVariableName"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * class MyClass {
+ *   void MyMethod() {
+ *     for (int var = 1; var &lt; 10; var++) {} // OK
+ *     for (int VAR = 1; VAR &lt; 10; VAR++) {} // violation, name 'VAR' must match
+ *                                           // pattern '^[a-z][a-zA-Z0-9]*$'
+ *     for (int i = 1; i &lt; 10; i++) {} // OK
```

Comment:
By default allowOneCharVarInForLoop is false.
So there should be violation. Am I right ?

Reply:
Well, this is why I thought it is weird. Here is the test result:

```
Jack@Jack MINGW64 ~/Documents/tmp (new-user)
$ cat TestLocalVariableName.java
class MyClass {
  void MyMethod() {
    for (int var = 1; var < 10; var++) {} // OK
    for (int VAR = 1; VAR < 10; VAR++) {} // violation, name 'VAR' must match
                                          // pattern '^[a-z][a-zA-Z0-9]*$'
    for (int i = 1; i < 10; i++) {} // OK
  }
}
Jack@Jack MINGW64 ~/Documents/tmp (new-user)
$ cat config.xml
<!DOCTYPE module PUBLIC
        "-//Checkstyle//DTD Checkstyle Configuration 1.3//EN"
        "https://checkstyle.org/dtds/configuration_1_3.dtd">
<module name="Checker">
  <module name="TreeWalker">
        <module name="LocalVariableName">
      <property name="severity" value="warning"/>
    </module>
  </module>
</module>
Jack@Jack MINGW64 ~/Documents/tmp (new-user)
$ java $RUN_LOCALE -jar checkstyle-8.18-all.jar -c config.xml TestLocalVariableName.java
Starting audit...
[WARN] C:\Users\asus\Documents\tmp\TestLocalVariableName.java:4:14: Name 'VAR' must match pattern '^[a-z][a-zA-Z0-9]*$'. [LocalVariableName]
Audit done.
```

Only name 'VAR' would have the violation.

Reply:
Please create issue on this, and let's keep in documentation what it right.
I am surprised that we have a bug in such simple Check.

Reply:
Done.